tests/test_input_damages.py

```diff
@@ -0,0 +1,1287 @@
+    """
+    ds_out_expected = xr.Dataset(
+        {
+            "histclim_rebased": (["year", "region"], np.full((2, 2), 2)),
+            "delta_rebased": (["year", "region"], np.full((2, 2), 0)),
+        },
+        coords={
+            "year": (["year"], [2010, 2099]),
+            "region": (["region"], ["ZWE.test_region", "USA.test_region"]),
+        },
+    )
+
+    d = os.path.join(
+        tmp_path, "labor_in", "batch6", "rcp45", "ACCESS1-0", "high", "SSP3"
+    )
+
+    ds_out_actual = calculate_labor_impacts(d, file_prefix, variable, val_type)
+    xr.testing.assert_equal(ds_out_expected, ds_out_actual)
+
+
+@pytest.mark.parametrize("out_format", ["return", "save"])
+def test_concatenate_labor_damages(
+    tmp_path,
+    econvars_fixture,
+    labor_in_val_fixture,
+    labor_in_histclim_fixture,
+    out_format,
+):
+    """
+    Test that concatenate_labor_damages correctly concatenates damages across batches
+    """
+    ds_out_expected = xr.Dataset(
+        {
+            "histclim_rebased": (
+                ["ssp", "rcp", "model", "gcm", "batch", "year", "region"],
+                np.float32(np.full((1, 2, 2, 2, 1, 2, 2), 2 * -1 * 1.273526)),
+            ),
+            "delta_rebased": (
+                ["ssp", "rcp", "model", "gcm", "batch", "year", "region"],
+                np.float32(np.full((1, 2, 2, 2, 1, 2, 2), 0)),
+            ),
+        },
+        coords={
+            "year": (["year"], [2010, 2099]),
+            "region": (["region"], ["ZWE.test_region", "USA.test_region"]),
+            "ssp": (["ssp"], ["SSP3"]),
+            "rcp": (["rcp"], ["rcp45", "rcp85"]),
+            "gcm": (["gcm"], ["ACCESS1-0", "GFDL-CM3"]),
+            "model": (["model"], ["IIASA GDP", "OECD Env-Growth"]),
+            "batch": (["batch"], ["batch9"]),
+        },
+    )
+
+    ds_out_actual = concatenate_labor_damages(
+        input_path=os.path.join(tmp_path, "labor_in"),
+        save_path=tmp_path,
+        ec_cls=econvars_fixture,
+    )
+
+    if out_format == "return":
+        xr.testing.assert_equal(ds_out_expected, ds_out_actual)
+
+    elif out_format == "save":
+        xr.testing.assert_equal(
+            ds_out_expected,
+            xr.open_dataset(os.path.join(tmp_path, "rebased_wage-levels_batch9.nc4")),
+        )
+
+
+def test_error_concatenate_labor_damages(
+    caplog,
+    econvars_fixture,
+    tmp_path,
+    labor_in_val_fixture,
+    labor_in_histclim_fixture,
+):
+    """
+    Test that concatenate_labor_damages returns an exception when any batch cannot be processed succesfully
+    """
+    os.makedirs(
+        os.path.join(tmp_path, "labor_in", "batch6", "rcp45", "CCSM4", "high", "SSP3")
+    )
+    concatenate_labor_damages(
+        input_path=os.path.join(tmp_path, "labor_in"),
+        save_path=tmp_path,
+        ec_cls=econvars_fixture,
+    )
+    assert "Error in batchbatch6" in caplog.text
```

Comment:
is there actually a typo in the error message that is being tested against?

Reply:
It is not a typo here, but a typo in the `concatenate_labor_damages` func. This can be fixed easily.

Reply:
Cool, not a big deal. Thanks!